DigitalOffice/Services/ProjectService/Repositories/ProjectRepository.cs

```diff
@@ -53,5 +54,37 @@ public Guid EditProjectById(DbProject dbProject)
 
             return dbProject.Id;
         }
+
+        public bool DisableWorkersInProject(WorkersIdsInProjectRequest request)
+        {
+            DbProject dbProject = dbContext.Projects
+                .FirstOrDefault(p => p.Id == request.ProjectId);
+
+            if (dbProject == null)
+            {
+                throw new NullReferenceException("Project with this Id does not exist");
+            }
+
+            foreach (Guid worker in request.WorkersIds)
+            {
+                DbProjectWorkerUser dbProjectWorker = dbProject.WorkersUsersIds?
+                    .Select(x => x)
+                    .FirstOrDefault(w => w.WorkerUserId == worker);
+
+                if (dbProject == null)
+                {
+                    throw new NullReferenceException("Worker with this Id does not exist");
+                }
+
+                dbProjectWorker.IsActive = false;
+
+                dbProject.WorkersUsersIds.Add(dbProjectWorker);
```

Comment:
What for? workerId already in this DbSet u already find it there. check line 70

Reply:
Fixed